lib/noop-set.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+
+module.exports = function noopSet () {
+  return {
+    [Symbol.iterator]: function * () {},
+    add () {},
+    delete () {},
+    has () { return true }
```

Comment:
Wouldn't the noop set be empty, and return `false` for all `.has()` calls?

Reply:
Returning `true` prevents new connections from being added to the set and thus avoids a branch and extra function invocation (`.add()`).